repo_manager/main.py

```diff
@@ -65,18 +68,41 @@ def main():  # noqa: C901
             if this_diffs is not None:
                 diffs[check_name] = this_diffs
 
-    actions_toolkit.debug(json_diff := json.dumps({}))
+    actions_toolkit.debug(json_diff := json.dumps(diffs))
     actions_toolkit.set_output("diff", json_diff)
 
     if inputs["action"] == "check":
         if not check_result:
+            actions_toolkit.info(inputs["repo_object"].full_name)
```

Comment:
@andrewthetechie - if you want to, you can remove these two lines I added...  Was just trying to get details on what differences were being flagged so I knew which module to investigate.

Reply:
also, I tried using the new debug methodology GitHub has: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging

But I was unable to see any debug messages...  Maybe action_toolkit debug method not compatible with the new changes to GitHub debugging?